docs/7.x/changelog.md

```diff
@@ -33,22 +33,70 @@ These versions are past their End of Support date, and no longer receive securit
 
 # Release Notes
 
+## 7.1 Releases
+
+7.1 is currently pre-release.
+
+### 7.1.0-alpha.5 (March 16, 2021)
+
+All changes listed are in comparison to 7.0.30 LTS.
+
+#### Improvements
+* Open source all Gravity code previously limited to enterprise customers! ([#2375](https://github.com/gravitational/gravity/pull/2375))
+* Add support for kubernetes 1.19 ([#2425](https://github.com/gravitational/gravity/pull/2425), [planet#825](https://github.com/gravitational/planet/pull/825), [storage-app#5](https://github.com/gravitational/storage-app/pull/5), [logging-app#76](https://github.com/gravitational/logging-app/pull/76), [monitoring-app#206](https://github.com/gravitational/monitoring-app/pull/206), [bandwagon#33](https://github.com/gravitational/bandwagon/pull/33))
+* Add an Ingress App to gravity ([#1435](https://github.com/gravitational/gravity/pull/1435))
+* Allow OpsCenter and the Web Installation wizard to be disabled in the cluster manifest ([#2371](https://github.com/gravitational/gravity/pull/2371))
+* Allow Logging, Monitoring, Ingress, Tiller, Storage, and Bandwagon applications to be disabled in the cluster manifest ([#2397](https://github.com/gravitational/gravity/pull/2397))
+* Expose additional logrange fields in logging-app ([#2369](https://github.com/gravitational/gravity/pull/2369), [logging-app#75](https://github.com/gravitational/logging-app/pull/75))
+* Allow Kubernetes High Availability mode to be enabled in the cluster manifest ([#2404](https://github.com/gravitational/gravity/pull/2404), [planet#821](https://github.com/gravitational/planet/pull/821))
+* Many scaleability tweaks for large clusters ([#2426](https://github.com/gravitational/gravity/pull/2426)) incuding:
+    * Introduce a hidden flag that eliminates workers from upgrade plan
+    * Introduce a hidden flag to allow parallel execution of certain upgrade phases
+    * Cache plan phases instead of re-querying Etcd each time
+    * Parallelize phases that run on multiple nodes, but do not affect the cluster (e.g. configuriation)
+    * Improve AWS event handler integration
+    * Add debug logging (including response times) to our internal API client
+    * Pre-emptively exit if an expand operation won't be accepted
+    * Allow multiple worker shrink/expand operations to run at the same time
+* Split `tele helm build` from `tele build` ([#1317](https://github.com/gravitational/gravity/pull/1317))
+* Bump Helm 2 version to 2.16.12 ([#2307](https://github.com/gravitational/gravity/pull/2307), [#2310](https://github.com/gravitational/gravity/pull/2310), [planet#789](https://github.com/gravitational/planet/pull/789))
+
+#### Removals
+* Removed support for Debian 8 ([#2210](https://github.com/gravitational/gravity/pull/2210))
+* Removed provisioner resources from the reference `telekube` application image ([#1561](https://github.com/gravitational/gravity/pull/1561))
+* Removed the Gravity's terraform provider ([#1622](https://github.com/gravitational/gravity/pull/1622))
```

Comment:
I'm not sure how consistent imperative mood has been used throughout the docs so what do you think of dropping the past tense?
```suggestion
* Remove support for Debian 8 ([#2210](https://github.com/gravitational/gravity/pull/2210))
* Remove provisioner resources from the reference `telekube` application image ([#1561](https://github.com/gravitational/gravity/pull/1561))
* Remove the Gravity's terraform provider ([#1622](https://github.com/gravitational/gravity/pull/1622))
```